docs/en/sql-reference/functions/array-functions.md

```diff
@@ -1794,6 +1794,52 @@ Return value type is always [Float64](../../sql-reference/data-types/float.md).
 │ 6   │ Float64                                                                                  │
 └─────┴──────────────────────────────────────────────────────────────────────────────────────────┘
 ```
+## arrayRemove
+
+- Returns the array after filtering out the elements that pass the given predicate.
+
+**Syntax**
+
+``` sql
+arrayRemove(array, predicate)
+```
+
+**Arguments**
+
+- `array` – Array.
+- `predicate` – Condition to remove elements from the array
```

Comment:
It should work without a predicate as well:

`arrayRemove(arr, value)`

Reply:
I've kept the working very similar to arrayFilter. Maybe we can define another function for that sort of a functioningn such as `arrayRemoveElement` ?

Reply:
We should do it exactly as in the specification: one function arrayRemove, two overloads, and a compatibility alias for Postgres.